sources/packages/backend/libs/utilities/src/system-users/system-users.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from "@nestjs/common";
```

Comment:
I believe that this service should be moved to the `@sims/services`.
The `@sims/utilities` is supposed to be consumed by any app/lib. If we create this here, the `@sims/utilities` will have a dependency on the `@sims/sims-db` then we no longer can consume the  `@sims/utilities` on `@sims/sims-db` otherwise we would be creating a circular dependency, don't we? Does it make sense?

Reply:
mm..ya make sense